tests/test_dataset_dict.py

```diff
@@ -46,13 +46,15 @@ def test_flatten(self):
             {"a": [{"b": {"c": ["text"]}}] * 10, "foo": [1] * 10},
             features=Features({"a": {"b": Sequence({"c": Value("string")})}, "foo": Value("int64")}),
         )
+        dset_split._format_columns = ["a.b.c", "foo"]
```

Comment:
As previously mentioned by @lhoestq, here it should also be `a` instead of `a.b.c` as that column is created in `flatten()`

```suggestion
        dset_split._format_columns = ["a", "foo"]
```